### PR TITLE
Scaffolds README

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/165827942)

## User Story
As a visitor of GitHub
In order to understand what the application does and why
I would like to have a descriptive README.md file

## Changes proposed in this PR
* Edits README.md with a pitch about the project, the testing tools used, author information, acknowledgements, license information, build information & deployment link section whenever it will be available.

### CODE REVIEW

File: README.md
Comment (zanenkn) [inline README.md:30]: should we add here `rails db:migrate` ? (i am not sure so feel free to disagree)

File: README.md
Comment (zanenkn) [inline README.md:16]: i think we should add user stories here

Comment (raptorf1) [reply README.md:16]: Yea, you are right. Forgot about them :P 
Added the section now. Will be adding them as soon as they are merged ;)

File: README.md
Comment (raptorf1) [reply README.md:30]: Right again. Added ;)

File: README.md
Comment (tochman) [reply README.md:16]: Not sure about that. I would explain the main features and link to PT. There's a danger that this document becomes too big. It's your call though.

Comment (raptorf1) [reply README.md:16]: That's a thought also. Thanks Thomas.

**Comment by faraznaeem** [comment]
You can also add your coverage badge in your REDAME, like this:
[Coverage Status](https://coveralls.io/github/CraftAcademy/cat-bnb?branch=development)

**Comment by raptorf1** [comment]
> You can also add your coverage badge in your REDAME, like this:
> [Coverage Status](https://coveralls.io/github/CraftAcademy/cat-bnb?branch=development)

Added ;)
Thanks!

**Review by leiter007 — COMMENTED** [review]
Think it looks good, no further comments!

**Comment by boamatule** [comment]
It looks well detailed. Nothing to add at the moment.

File: README.md
Comment (zanenkn) [inline README.md]: Shall we deploy?

Comment (raptorf1) [reply README.md]: deployed by dear Zane :)

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**
